src/gui/optionsdlg.cpp

```diff
@@ -445,25 +454,19 @@ void OptionsDialog::changePage(QListWidgetItem *current, QListWidgetItem *previo
 
 void OptionsDialog::loadWindowState()
 {
-    const Preferences* const pref = Preferences::instance();
-
-    resize(pref->getPrefSize(this->size()));
+    Utils::Gui::resize(this, Preferences::instance()->getPrefSize());
 }
 
 void OptionsDialog::loadSplitterState()
 {
-    const Preferences* const pref = Preferences::instance();
+    const QStringList sizesStr = Preferences::instance()->getPrefHSplitterSizes();
 
-    const QStringList sizes_str = pref->getPrefHSplitterSizes();
-    QList<int> sizes;
-    if (sizes_str.size() == 2) {
-        sizes << sizes_str.first().toInt();
-        sizes << sizes_str.last().toInt();
-    }
-    else {
-        sizes << 116;
-        sizes << m_ui->hsplitter->width() - 116;
-    }
+    // width has been modified, use height as width reference instead
+    const int width = Utils::Gui::scaledSize(this
+        , (m_ui->tabSelection->item(TAB_UI)->sizeHint().height() * 2));
+    QList<int> sizes { width, (m_ui->hsplitter->width() - width) };
```

Comment:
According to coding guidelines (**d. Brace enclosed initializers**) there shouldn't be surrounding spaces inside curly braces in case of initializers.
(There are some other places with similar issue)

Reply:
Sorry I can't remember it correctly every time, I should have fixed all instances now.